core/extract/extract.py

```diff
@@ -20,17 +20,28 @@ def main():
     config_helper.validate_config_file_for_extract(config)
 
     # UID for extract logs
-    cluster = cluster_dict(config["source_cluster_endpoint"])
     extract_start_timestamp = datetime.datetime.now(tz=datetime.timezone.utc)
     id_hash = hashlib.sha1(
         extract_start_timestamp.isoformat().encode("UTF-8")
     ).hexdigest()[:5]
-    if config.get("tag", "") != "":
-        extract_id = f'{extract_start_timestamp.isoformat()}_{cluster.get("id")}_{config["tag"]}_{id_hash}'
+    if config.get('source_cluster_endpoint'):
```

Comment:
Think you need a similar validation to the one you have on tag over here.

Reply:
Done